app/views/appointments/_appointment.html.erb

```diff
@@ -28,7 +28,17 @@
         </div>
       </div>
     <% end %>
-    <br/>
+
+    <div class="row mt-4">
+      <div class="col">
+        <strong>Imprima ou salve esta página para sua conferência.</strong>
+      </div>
+    </div>
+    <div class="row mb-4">
+      <div class="col text-muted">
+        Código do agendamento: <%= "U#{appointment.ubs_id}P#{appointment.patient_id}A#{appointment.id}" %>
```

Comment:
Added "codigo do agendamento" so we can have a reference in case there are dupes or issues.